packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -120,14 +119,11 @@
     --search-bar-background: var(--c-white);
   }
   @include for-desktop {
-    --header-logo-height: 2.5rem;
     --header-logo-margin: 0;
-    --header-sticky-container-background: var(--c-white);
     --header-container-height: 5rem;
     --header-container-justify-content: space-between;
-    --header-container-padding: 0 var(--spacer-extra-big);
+    --header-container-padding: 0 calc(var(--spacer-sm) * 6);
```

Comment:
```suggestion
    --header-container-padding: 0;
```
we don't need padding on new designs :) 
![Zrzut ekranu 2020-03-17 o 15 07 03](https://user-images.githubusercontent.com/12138170/76864178-f7194900-6860-11ea-8f48-907add681a3f.png)